src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer smallBulldozer = new Bulldozer();
+    private Bulldozer mediumBulldozer = new Bulldozer();
+    private Bulldozer bigBulldozer = new Bulldozer();
+    private List<Bulldozer> bulldozerList = List.of(bigBulldozer, mediumBulldozer, smallBulldozer);
```

Comment:
`to produce` means to create new objects each time when we are trying to get them, so please move this logic to the `get` method
```suggestion
```